pkg/controller/volume/persistentvolume/binder_test.go

```diff
@@ -517,6 +519,76 @@ func TestSync(t *testing.T) {
 			newClaimArray("claim13-5", "uid13-5", "1Gi", "volume13-5", v1.ClaimBound, nil, annBoundByController, annBindCompleted),
 			noevents, noerrors, testSyncClaim,
 		},
+
+		// PVC with VolumeMode
```

Comment:
Can you also add test cases for prebound PVC and prebound PV?

Reply:
yep, done